e2e/stories/dataset/aggregations.e2e.ts

```diff
@@ -0,0 +1,93 @@
+import { ISetField } from "../../management";
+import { cleaning, DEFAULT_DATASET, dom, init } from "../../teardowns";
+
+jest.setTimeout(15000);
+
+const fields: ISetField[] = [{
+  name: "name",
+  type: "string"
+}, {
+  name: "age",
+  type: "integer",
+}];
+
+const data = [
+  { name: "Boris", age: 18 },
+  { name: "Anton", age: 42 },
+  { name: "Olga", age: 24 },
+  { name: "Slava", age: 33 },
+];
+
+describe("Dataset aggregation functions", () => {
+  beforeAll(async () => {
+    await init(DEFAULT_DATASET.NAME, fields);
+    await dom.dataset(DEFAULT_DATASET.NAME).insert(data).execute();
+  });
+
+  afterAll(async () => cleaning());
+
+  it("should select max value", (done) => {
+    dom.dataset(DEFAULT_DATASET.NAME)
+      .select()
+      .fields({ fn: "max", col: "age"})
+      .subscribe({
+        next: (result) => {
+          expect(result).toEqual([{ max: 42 }]);
+          done();
+        },
+        error: done,
```

Comment:
`error: done.fail`

Reply:
Hm, actually doesn't matter because `done` with any not empty argument will fail the test